web-admin/src/routes/[organization]/[project]/explore/[dashboard]/+page.svelte

```diff
@@ -118,6 +118,21 @@
       !!exploreResponse.explore?.meta?.reconcileError;
     return isMetricsViewErrored || isExploreErrored;
   }
+
+  let reconcilingForFirstTime: boolean | undefined;
+  $: if ($explore.isSuccess) {
+    const newReconcilingForFirstTime = isDashboardReconcilingForFirstTime(
+      $explore.data,
+    );
+    if (
+      reconcilingForFirstTime !== undefined &&
+      newReconcilingForFirstTime !== undefined &&
+      reconcilingForFirstTime !== newReconcilingForFirstTime
+    ) {
```

Comment:
An explanatory variable would make this more clear. Also, would this simpler condition work?
```
const justFinishedFirstReconcile = reconcilingForFirstTime === true && newReconcilingForFirstTime === false
if (justFinishedFirstReconcile) void invalidate(`explore:${exploreName}`);
```

```

Reply:
Sure adding a variable. That new condition wont work when switching from `false` to `true` when redeployed. Probably good to keep it consistent.